### PR TITLE
feat: Add size and tone props to ma-option

### PR DESCRIPTION
**Pull request by carlotamolintr**
These two props solve the text alignment issue in call me component. 
image

Cause for an specific size text will be a margin-top to align the text with the checkbox.

Current issue in kali:

We are duplicating the ma-text component in call me, the first one is being used by the ma-option and the second one is being used to apply size and tone. The second one is making issues with the positioning so with this PR we could remove one

### CODE REVIEW

**Review by afontcu — APPROVED** [review]
Looks good to me! 🚀

**Review by edufarre — APPROVED** [review]

File: src/components/MaOption/MaOptionCheckbox.css [hunk 1]
```diff
@@ -16,9 +16,23 @@
...
+.ma-selector-small {
+  &.ma-option--checkbox .description {
+    margin-top: 3px;
+  }
+}
+.ma-selector-medium {
+  &.ma-option--checkbox .description {
+    margin-top: 2px;
+  }
+}
```
Comment (Samitier) [inline src/components/MaOption/MaOptionCheckbox.css:35]: this nesting is useless no? And `ma-selector-xxx` is always in `ma-option--checkbox` so why not:
```suggestion
.ma-selector-xsmall .description {
  margin-top: 4px;
}
.ma-selector-small .description {
  margin-top: 3px;
}
.ma-selector-medium  .description {
  margin-top: 2px;
}
```

File: src/components/MaOption/MaOptionRadio.css [hunk 2]
```diff
@@ -10,9 +10,23 @@
...
+.ma-selector-small {
+  &.ma-option--radio .description {
+    margin-top: 3px;
+  }
+}
+.ma-selector-medium {
+  &.ma-option--radio .description {
+    margin-top: 2px;
+  }
+}
```
Comment (Samitier) [inline src/components/MaOption/MaOptionRadio.css:27]: same, useless nesting & redundant `ma-option--radio` selector, no?

**Review by Samitier — APPROVED** [review]
just a tiny stuff with the css selectors and lgtm!

File: src/components/MaOption/MaOptionRadio.css (hunk 2 above)
Comment (carlotamolintr) [reply src/components/MaOption/MaOptionRadio.css]: nesting removed!

File: src/components/MaOption/MaOptionCheckbox.css (hunk 1 above)
Comment (carlotamolintr) [reply src/components/MaOption/MaOptionCheckbox.css]: nesting removed!

**Pull request merged**